src/components/TablePreview/TablePreview.story.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ *  Component Story Format (CSF) is now encouraged.
+ *  More info here: https://storybook.js.org/docs/react/api/csf
+ */
+
+//import { action } from "@storybook/addon-actions";
+import { Story } from "@storybook/react";
+import React, { ComponentProps, ReactElement } from "react";
+
+import TablePreview from "./TablePreview";
+
+export default {
+    title: "TSX/TablePreview",
+    component: TablePreview,
+};
+
+const TablePreviewStory: Story = (
+    args: ComponentProps<typeof TablePreview>,
+): ReactElement => {
+    // Shared props may be passed here
+    return <TablePreview {...args} />;
+};
+
+// Story specific props are passed here
+const Enabled = TablePreviewStory.bind({});
+
+export { Enabled };
```

Comment:
Looks good! 

Cambiemos este variable name a algo mas meaningful, btw el nombre que uno le pone al exported story, va a ser el nombre que sale en storybook, ahorita sale así: 
![image](https://user-images.githubusercontent.com/52969178/224775084-ddb4be7b-1b35-4ada-8da6-6f1b90391276.png)

Cambiemoslo a algo como `BaseTable` o `HtmlTable` or something like that, your call!

Reply:
Made the change ref to HtmlTable